storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/MetricCollector.java

```diff
@@ -0,0 +1,137 @@
+    private final Sensor completeMultipartUploadRequests;
+    private final Sensor abortMultipartUploadRequests;
+
+    MetricCollector() {
+        final JmxReporter reporter = new JmxReporter();
+
+        metrics = new org.apache.kafka.common.metrics.Metrics(
+            new MetricConfig(), List.of(reporter), Time.SYSTEM,
+            new KafkaMetricsContext("aiven.kafka.server.tieredstorage.s3")
+        );
+        final String metricGroup = "s3-metrics";
+
+        getObjectRequests = metrics.sensor("get-object-requests");
+        getObjectRequests.add(
+            metrics.metricName("get-object-requests-rate", metricGroup), new Rate());
+        getObjectRequests.add(
+            metrics.metricName("get-object-requests-total", metricGroup), new CumulativeCount());
+
+        putObjectRequests = metrics.sensor("put-object-requests");
+        putObjectRequests.add(
+            metrics.metricName("put-object-requests-rate", metricGroup), new Rate());
+        putObjectRequests.add(
+            metrics.metricName("put-object-requests-total", metricGroup), new CumulativeCount());
+
+        deleteObjectRequests = metrics.sensor("delete-object-requests");
+        deleteObjectRequests.add(
+            metrics.metricName("delete-object-requests-rate", metricGroup), new Rate());
+        deleteObjectRequests.add(
+            metrics.metricName("delete-object-requests-total", metricGroup), new CumulativeCount());
+
+        createMultipartUploadRequests = metrics.sensor("create-multipart-upload-requests");
+        createMultipartUploadRequests.add(
+            metrics.metricName("create-multipart-upload-requests-rate", metricGroup), new Rate());
+        createMultipartUploadRequests.add(
+            metrics.metricName("create-multipart-upload-requests-total", metricGroup), new CumulativeCount());
+
+        uploadPartRequests = metrics.sensor("upload-part-requests");
+        uploadPartRequests.add(
+            metrics.metricName("upload-part-requests-rate", metricGroup), new Rate());
+        uploadPartRequests.add(
+            metrics.metricName("upload-part-requests-total", metricGroup), new CumulativeCount());
+
+        completeMultipartUploadRequests = metrics.sensor("complete-multipart-upload-requests");
+        completeMultipartUploadRequests.add(
+            metrics.metricName("complete-multipart-upload-requests-rate", metricGroup), new Rate());
+        completeMultipartUploadRequests.add(
+            metrics.metricName("complete-multipart-upload-requests-total", metricGroup), new CumulativeCount());
+
+        abortMultipartUploadRequests = metrics.sensor("abort-multipart-upload-requests");
+        abortMultipartUploadRequests.add(
+            metrics.metricName("abort-multipart-upload-requests-rate", metricGroup), new Rate());
+        abortMultipartUploadRequests.add(
+            metrics.metricName("abort-multipart-upload-requests-total", metricGroup), new CumulativeCount());
```

Comment:
Are these `-rate` and `-total` common suffices for every sensor here? Or can there be others in future? If no can we create a method that initializes the sensor and adds these metrics instead of this duplication. For example like this:
```suggestion
        getObjectRequests = getSensor(metricGroup, "get-object-requests");
        putObjectRequests = getSensor(metricGroup, "put-object-requests");
        deleteObjectRequests = getSensor(metricGroup, "delete-object-requests");
        createMultipartUploadRequests = getSensor(metricGroup, "create-multipart-upload-requests");
        uploadPartRequests = getSensor(metricGroup, "upload-part-requests");
        completeMultipartUploadRequests = getSensor(metricGroup, "complete-multipart-upload-requests");
        abortMultipartUploadRequests = getSensor(metricGroup, "abort-multipart-upload-requests");
    }

    private Sensor getSensor(final String metricGroup, final String name) {
        final Sensor sensor = metrics.sensor(name);
        sensor.add(metrics.metricName(name + "-rate", metricGroup), new Rate());
        sensor.add(metrics.metricName(name + "-total", metricGroup), new CumulativeCount());
        return sensor;
    }
```
or similar

Reply:
Yes, this would be shorter. Did this